src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -104,14 +105,16 @@ export class CalcitePickList {
   //
   // --------------------------------------------------------------------------
 
+  connectedCallback() {
+    sharedListMethods.initialize.call(this);
```

Comment:
Moving these back to connectedCallback was intentional. Once the logic is modified to use props instead of attributes, it shoouldn't need to be delayed until after first-render. #400